packages/documentation/src/sidebar.js

```diff
@@ -94,6 +94,35 @@ module.exports = {
           name: 'Site structure',
           items: [],
         },
+        {
+          name: 'Front End Standards',
```

Comment:
You can see from the images that this is under architecture > front end standards.